domain/src/main/java/camp/nextstep/edu/calculator/domain/Expression.kt

```diff
@@ -0,0 +1,38 @@
+package camp.nextstep.edu.calculator.domain
+
+class Expression(formulaString: String?) {
+
+    private val formulas: List<String>
```

Comment:
```suggestion
    val formulas: List<String>
        private set
```

코틀린에서 멤버변수는 프로퍼티의 특징을 가지고있습니다.
getExpressions를 따로 만들지 않고, 이미 있는 getter를 사용해보는 쪽으로 코드를 변경해보시면 좋을 것 같습니다 :)

Reply:
다음 미션에 반영해보겠습니다.